File: 2.0/java/campaigns/findCampaignSounds.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CampaignSound;
import com.callfire.api.client.api.campaigns.model.request.FindSoundsRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        FindSoundsRequest request = FindSoundsRequest.create()
            .filter("name")
            .includeArchived(true)
            .includePending(true)
            .includePending(true)
            .offset(0L)
            .limit(10L)
            .fields("items(id,name,callback)")
            .build();
        Page<CampaignSound> campaignSounds = callfireClient.campaignSoundsApi().find(request);
    }
}

File: 2.0/csharp/campaigns/findCampaignSounds.cs
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model.Request;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        FindSoundsRequest request = new FindSoundsRequest
        {
            Filter = "name",
            IncludeArchived = true,
            IncludePending = true,
            IncludeScrubbed = true,
            Offset = 0,
            Limit = 10,
            Fields = "items(id,name,callback)"
        };
        Page<CampaignSound> campaignSounds = client.CampaignSoundsApi.Find(request);
    }
}
